hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -206,6 +206,24 @@ podman build -t "${SRIOV_NETWORK_CONFIG_DAEMON_IMAGE}" -f "${root}/Dockerfile.sr
 echo "## build webhook image"
 podman build -t "${SRIOV_NETWORK_WEBHOOK_IMAGE}" -f "${root}/Dockerfile.webhook" "${root}"
 
+echo "## wait for the all cluster to be stable"
+MAX_RETRIES=20
+DELAY_SECONDS=10
+retries=0
+until [ $retries -ge $MAX_RETRIES ]; do
+  if [ $(kubectl get clusteroperator --no-headers | awk '{print $3}' | grep True | wc -l) -eq 33 ]; then
```

Comment:
nit: please add comment what it is we are checking here

Reply:
done :)